data/transactions/stateproof.go

```diff
@@ -18,50 +18,42 @@ package transactions
 
 import (
 	"github.com/algorand/go-algorand/crypto"
-	"github.com/algorand/go-algorand/crypto/compactcert"
+	"github.com/algorand/go-algorand/crypto/stateproof"
 	"github.com/algorand/go-algorand/data/basics"
+	"github.com/algorand/go-algorand/data/stateproofmsg"
 	"github.com/algorand/go-algorand/protocol"
 )
 
-// CompactCertTxnFields captures the fields used for compact cert transactions.
-type CompactCertTxnFields struct {
+// StateProofTxnFields captures the fields used for stateproof transactions.
+type StateProofTxnFields struct {
 	_struct struct{} `codec:",omitempty,omitemptyarray"`
 
-	CertRound basics.Round             `codec:"certrnd"`
-	CertType  protocol.CompactCertType `codec:"certtype"`
-	Cert      compactcert.Cert         `codec:"cert"`
+	StateProofIntervalLatestRound basics.Round            `codec:"sprnd"`
```

Comment:
I wonder if we really need this field.
the same value can be found on Message->LastAttestedRound